.github/workflows/release.yaml

```
@@ -79,18 +81,28 @@ jobs:
           fork-user: prql-bot
 
   build-deb-package:
-    # TODO: currently this doesn't publish it as an artifact; would be a welcome contribution
-    # to add that (and eventually as a release asset)
-    runs-on: ubuntu-latest
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: false
+      matrix:
+        include:
+          - os: ubuntu-latest
+            target: x86_64-unknown-linux-musl
+          - os: ubuntu-latest
+            target: aarch64-unknown-linux-musl
```

Obviously only Linux is the OS here, so I think we can specify only `target` without using include.

Something like:

```suggestion
      matrix:
         target:
           - x86_64-unknown-linux-musl
           - aarch64-unknown-linux-musl
```